burr/integrations/persisters/postgresql.py

```diff
@@ -0,0 +1,234 @@
+try:
```

Comment:
Let's stick this in integrations actually? Or at least use the same pattern here: https://github.com/DAGWorks-Inc/burr/blob/main/burr/tracking/client.py#L30. It automates away the message so they're all the same.

Reply:
it is in integrations, but under persisters.